readinglist/__init__.py

```diff
@@ -55,7 +55,6 @@ def set_auth(config):
 
     config.set_authorization_policy(authz_policy)
     config.set_authentication_policy(authn_policy)
-    config.set_default_permission('readwrite')
```

Comment:
Instead of removing the line, would you mind leaving it with a XXX comment that it should be restored once the patch is merged and released upstream ?